runtime/near-vm-runner/src/tests.rs

```diff
@@ -55,6 +55,7 @@ fn create_context(input: Vec<u8>) -> VMContext {
 /// Includes hard-coded value for runtime parameter values
 /// `wasm_contract_loading_base` and `wasm_contract_loading_bytes` which would
 /// have to be updated if they change in the future.
+#[cfg(target_arch = "x86_64")]
```

Comment:
I'd allow(unused) -- it semes like this'll fail on mac with `--feature protocol_feature_fix_contract_loading_cost`

Reply:
Ah good spot!  now updated.